src/kohlrahbi/unfoldedahb/unfoldedahbtable.py

```diff
@@ -27,6 +27,38 @@
 _segment_group_pattern = re.compile(r"^SG\d+$")
 
 
+def _lines_are_equal_when_ignoring_guid(line1: AhbLine, line2: AhbLine) -> bool:
+    """
+    returns true iff the line1 and line2 are equal except for their guid
+    """
+    line1_copy = copy.deepcopy(line1)
+    line2_copy = copy.deepcopy(line2)
+    line1_copy.guid = None
+    line2_copy.guid = None
+    return line1_copy == line2_copy
```

Comment:
uh nice. great that the AhbLine class supports is_equal out of the box :)

Reply:
well, that's one purpose of attrs, pydantic, dataclasses and alike